docs/lbl/lbl.rst

```diff
@@ -696,6 +691,27 @@ for an example, which can be run with (you will need the CDSD-HITEMP database in
 
     pytest radis/test/lbl/test_broadening.py -m "test_abscoeff_continuum"
 
+Choose the right wavenumber grid
+--------------------------------
+
+``wstep`` determines the wavenumber grid's resolution. Smaller the value, higher the resolution and
+vice-versa. By default **radis** uses ``wstep=0.01``. You can manually set the ``wstep`` value
+in :py:func:`~radis.lbl.calc.calc_spectrum` and :py:class:`~radis.lbl.factory.SpectrumFactory`.
+To get more accurate result you can further reduce the value, and to increase the performance you can increase the value.
+
+Based on ``wstep``, it will determine the number of gridpoints per linewidth.
+To make sure that there are enough gridpoints, Radis will raise an Accuracy Warning :py:meth:`~radis.lbl.broadening.BroadenFactory._check_accuracy`
+if number of gridpoints are less than :py:data:`~radis.params.GRIDPOINTS_PER_LINEWIDTH_WARN_THRESHOLD` and raises an Accuracy Error
+if number of gridpoints are less than :py:data:`~radis.params.GRIDPOINTS_PER_LINEWIDTH_ERROR_THRESHOLD`.
+
+From ``0.9.30`` a new mode ``wstep='auto'`` has been added which directly computes the optimum value of ``wstep``
+ensuring both performance and accuracy. It is ensured that there are slightly more or less than :py:data:`~radis.params.GRIDPOINTS_PER_LINEWIDTH_WARN_THRESHOLD`
+points for each linewidth.
+
+.. Note::
```

Comment:
I think capitalized won't work. You can directly accept the suggestion below :

```suggestion
.. note::
```

Reply:
`Note` also works fine.
![Screenshot from 2021-06-04 17-20-03](https://user-images.githubusercontent.com/44584067/120797286-73c53f80-c559-11eb-8ea1-0071bee8de43.png)